packages/cli/src/utils/ui.ts

```diff
@@ -5,15 +5,25 @@ import { colorPalette } from '../constants';
 import { capitalize } from './format';
 import { isVitest } from './';
 
+interface InfoOptions {
+  header?: boolean;
+  margin?: boolean;
+}
+
+interface ErrorOptions {
+  header?: boolean;
+  margin?: boolean;
+}
+
 const noopProgressBar = {
-  increment: () => {},
-  setTotal: (_n: number) => {},
-  stop: () => {},
+  increment: () => { },
+  setTotal: (_n: number) => { },
+  stop: () => { },
```

Comment:
thought: I've noticed a couple of formatting changes like these. Although they were mostly for the better, I'm not a fan of formatting changes "polluting" the diff of a PR. Maybe we should think about using a standardized Prettier (or oxfmt) config?